sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md

```diff
@@ -2,5 +2,75 @@
 
 ## 1.0.0-preview.1
 
-### New Features
-- Initial preview of Azure Management KeyVault SDK based on Azure.Core
\ No newline at end of file
+This is public preview version based on Azure Core. This version uses a next-generation code generator that introduces important new features.
+
+### General New Features
+
+    - Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET
+    - Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing
+    - Support uniform telemetry across all languages
+
+### Migration from Previous Version of Azure Management SDK
+
+#### Package Name
+The package name has been changed from `Microsoft.Azure.Management.KeyVault` to `Azure.Management.KeyVault`
+
+#### Create Key Vault
+
+Before
+```csharp
+using Microsoft.Azure.Management.KeyVault;
+using Microsoft.Azure.Management.KeyVault.Models;
+
+var keyVaultManagementClient = new KeyVaultManagementClient(credentials);
```

Comment:
should we add how we initialize credentials? 

eg. var credentials = DefaultAzureCredentials()